### PR TITLE
Create fields frontend

### PR DESCRIPTION
**Pull request by internettrans**
image

### CODE REVIEW

File: frontend/Utils/flaxFetch.tsx [hunk 1]
```diff
@@ -15,15 +15,9 @@ export function flaxFetch<ResponseDataType = object>(
     if (r.ok) {
       if (r.status === 204) {
         return;
-      } else if (
-        r.headers["content-type"] &&
```
Comment (internettrans) [inline frontend/Utils/flaxFetch.tsx:19]: This was a bug in flaxFetch - `r.headers['content-type']` was never defined in Chrome.

File: models/defaults.d.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export interface DefaultModelAttrs {
```
Comment (internettrans) [inline models/defaults.d.ts:1]: I tried for a while to find a type inside of sequelize that adds the createdAt/updatedAt/id properties to objects returned from the database, but couldn't find one. So I ended up creating this type.

File: webpack.config.cjs [hunk 3]
```diff
@@ -40,4 +40,9 @@ module.exports = {
     ],
   },
   plugins: [new MiniCssExtractPlugin()],
+  devServer: {
+    headers: {
+      "Access-Control-Allow-Origin": "*",
```
Comment (internettrans) [inline webpack.config.cjs:45]: I don't remember why I added this, but this configuration allows CORS requests to the dev server

File: backend/Fields/BatchPostFields.ts [hunk 4]
```diff
@@ -23,7 +23,7 @@ router.post<Params, ResponseBody, RequestBody>(
 
     const { nounId } = req.params;
 
-    let noun: NounAttributes;
+    let noun: NounModel | null;
```
Comment (rhiannonstanford) [inline backend/Fields/BatchPostFields.ts:26]: Did you mean to include a single pipe in the type declaration here?

**Review by rhiannonstanford — APPROVED** [review]

File: backend/Fields/BatchPostFields.ts (hunk 4 above)
Comment (internettrans) [reply backend/Fields/BatchPostFields.ts:26]: Yeah in typescript the single pipe is used to mean "or"

**Pull request merged**